src/mouse-keyboard-vr-display.js

```diff
@@ -161,6 +164,28 @@ MouseKeyboardVRDisplay.prototype.onMouseUp_ = function(e) {
   this.isDragging_ = false;
 };
 
+
+MouseKeyboardVRDisplay.prototype.requestPointerLock_ = function(element) {
+  // from pointerlock controls in three.js
+  // - https://github.com/mrdoob/three.js/blob/master/examples/misc_controls_pointerlock.html
+  element.requestPointerLock = element.requestPointerLock || element.mozRequestPointerLock || element.webkitRequestPointerLock;
+  if ( /Firefox/i.test( navigator.userAgent ) ) {
```

Comment:
i tested on chrome stable and firefox stable on macosx, and it worked. This is a code specific to desktop. i dont have IE unfortuntatly.

@cvan can you explain which part is not reliable ?